modules/ROOT/pages/advanced-typography.adoc

```diff
@@ -0,0 +1,60 @@
+= Advanced Typography plugin
+:navtitle: Advanced Typography
+:description: Automatically or interactively replace typewriter-centric characters with characters preferred by typesetters.
+:description_short: Auomatically or interactively typeset {productname} documents.
+:keywords: plugin, typography, typesetting, smart quotes, dashes, em-dash, en-dash, quotation marks
+:pluginname: Advanced Typography
+:plugincode: typography
+:plugincategory: premium
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.3v.adoc[]
+
+The {pluginname} plugin, depending on its setting, automatically or interactively replaces common typewriter-style characters with their professional typesetter’s equivalents.
+
+For example, if English-language typography rules are set, the {pluginname} plugin can automatically replace the double-hyphen (`--`) with an em-dash (`—`). And it can automatically replace the teardrop apostrophe (`'`) with opening and closing quote marks (`‘` and `’`).
+
+NOTE: The {pluginname} plugin rules are sourced from the https://github.com/typograf/typograf[typograf] library.
+
+== Interactive example
+
+liveDemo::{plugincode}[]
+
+
+== Basic setup
+
+To add the {pluginname} plugin to the editor, add `{plugincode}` to the `plugins` option in the editor configuration.
+
+NOTE: The {pluginname} plugin requires a `language` be configured.
+
+For example:
+
+[source,js,subs="attributes+"]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: '{plugincode}',
+  toolbar: '{plugincode}',
+  language: 'en_US' // This is required to configure the default language used by typography.
+});
+----
+
+// Delete everything not require/relevant from this point on.
+
+== Options
```

Comment:
@brian-forte Below options needs to be documented -

- typography_default_lang
- typography_langs

Test Fiddle used by QA team  for reference  - https://fiddle.tiny.cloud/uqiaab/58